source/_patterns/molecules/interactive/read-more/read-more.twig

```diff
@@ -0,0 +1,14 @@
+<div class="jcc-read-more">
+  <div id="target" class="jcc-read-more__content">  
```

Comment:
@claudioccm Looks like you have left over source directories?  This file and the Sass above it should not be in here: Note the directory name is `molecules` not `02-molecules`. Looks like you might to rebase.